### ISSUE TITLE
A11y_VisualStudioCodeClient_Editor_ Screen reader: Voiceover is announcing same information for both the suggestions for exclamation mark(!)

### ISSUE THREAD

**Issue by BhanuPrakashGit** [microsoft/vscode#180116]
**GitHub Tags:**
#A11yMAS; #A11yTCS; #A11ySev2; #Device; #Visual Studio Code Client; #BM-VisualStudioCodeClient-MAC-Oct2022; WCAG1.3.1; #Voiceover; #Screenreader.

### Environment and OS details:
Visual Studio Code Insiders
Screen reader: Voiceover

### Repro Steps:

1. Open Visual studio code editor
2. Create a HTML file.
3. Type (!) exclamation mark .
4. Suggestions will appear.
5. Turn on Voiceover
6. Verify that Voiceover is announcing same information for both the suggestions for exclamation mark(!).

### Actual Result:
Voiceover is announcing same information for both the suggestions for exclamation mark(!).

### Expected Result:
Voiceover should announce information related to the suggestion. Voiceover should not announce the same information for both the suggestions.

### Attachment:
![198011683-5ba9bf72-3fd3-4932-a4a5-e74544fc3071](https://user-images.githubusercontent.com/104351566/232459451-69074c5b-f697-49ff-910b-7fe9285efa1f.gif)
![198995885-c7df4b59-d8cc-4acf-bfd5-23849e616b7f](https://user-images.githubusercontent.com/104351566/232459482-f21d427f-176e-4ac1-af52-2d492164a10d.gif)

**Comment by BhanuPrakashGit** [comment]
Reference bug: https://github.com/Microsoft/vscode/issues/164700#issue-1423847755

**Comment by BhanuPrakashGit** [comment]
GithubBug:#Regressed:04-17-23
Logged IRT bug against VoiceOver.
